src/main/java/org/broadinstitute/consent/http/cloudstore/GCSService.java

```diff
@@ -67,23 +67,24 @@ public Bucket getRootBucketWithMetadata() {
     }
 
     /**
-     * Store an input stream as a Blob
+     * Store an input stream as a Blob with the file name as the blob name
      *
      * @param content InputStream content
      * @param mediaType String media type
-     * @param fileName String file name
+     * @param bucketName String unique name of the bucket
      * @return BlobId of the stored document
      * @throws IOException Exception when storing document
      */
-    public BlobId storeDocument(InputStream content, String mediaType, String fileName)
+    public BlobId storeDocument(InputStream content, String mediaType, String bucketName)
```

Comment:
wanted to make it clearer that the fileName was just what is used as the name of the bucket. This change was made because I was passing in UUIDs instead of the file name. I pass in UUIDs so that there are no conflicts if users upload two files with the exact same name.

Reply:
Would it be better to change the type to UUID and the name to bucketUUID?

Reply:
Right now, it is used in other places. There is another ticket in flight which would make that change possible - https://broadworkbench.atlassian.net/browse/DUOS-2184 - I believe within this ticket it would be possible to make that a UUID type.

Reply:
This is a great find, by the way.  I'm digging in a bit further to make sure I understand the full repercussions of what " I pass in UUIDs so that there are no conflicts if users upload two files with the exact same name" implies about the existing codebase.

Reply:
IIUC, this is intended to store the parent bucket that the file is stored in. That's not a UUID presently, it's a plain string. Look at your local dev config for an example:
```
googleStore:
  password: /etc/consent-account.json
  endpoint: https://storage.googleapis.com/
  bucket: broad-dsde-dev-consent
```

Reply:
> wanted to make it clearer that the fileName was just what is used as the name of the bucket.

I don't think this is actually true ... bucket location is configured separately from the files that are named inside the bucket. Partial screen cap of our current dev bucket:

<img width="785" alt="Screen Shot 2022-11-16 at 12 47 42 PM" src="https://user-images.githubusercontent.com/116679/202255217-de2a9f24-3cbf-468c-812b-d7288a351e33.png">

Reply:
I just realized some point of confusion here: I meant to name this variable `blobName`. The variable is the name of the blob, which is why it's a UUID. The bucket name itself is `broad-dsde-dev-consent` always.

Reply:
Did a little test to put my mind at ease on this one.  I created two DARs, both requiring a COL agreement and uploaded two files (same name, different contents) and saw that the files are properly differentiated for return.

Reply:
DataAccessRequestResourceVersion2.java line #428 does the lifting to keep files uniquely named.  Now that I better understand things, do you really want to name this "blobId"?

Reply:
Oh, I see. I had gotten confused with the way the pathing had worked. Sure, we can make this a UUID type.